ts/screens/modal/IdentificationModal.tsx

```diff
@@ -330,6 +338,7 @@ const IdentificationModal = () => {
       transparent
       onRequestClose={onRequestCloseHandler}
     >
+      {Platform.OS === "ios" && <StatusBar barStyle={"light-content"} />}
```

Comment:
Actually in the previous identification modal the status bar was hidden, I missed it. What about hide it again? What do you think?

Reply:
I don't think it's a good idea, because even the official number pad view used in iOS doesn't hide the status bar. Sure, it provides a cleaner view, but at what cost?

Reply:
I don't have memory of the rationale about the old modal. But I guess that the more this modal is clean the more the focus of the user is. What do you think?

Reply:
Okay! 👍